catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/query/data/model/QueryBasic.java

```diff
@@ -123,7 +128,11 @@ public QueryBasic(Metacard metacard) {
     this.sources = getAttributeValues(metacard, QUERY_SOURCES, String.class);
 
     Class<Map<String, String>> clazz = (Class<Map<String, String>>) (Class) Map.class;
-    this.sorts = getAttributeValues(metacard, QUERY_SORTS, clazz);
+    this.sorts =
+        getAttributeValues(metacard, QUERY_SORTS, String.class)
+            .stream()
+            .map(sortStr -> GSON.fromJson(sortStr, clazz))
```

Comment:
should we catch the jsonsyntaxexception so that if bad data gets in the system somehow it does not fail it everytime someone tries to read that particular record?